Tests/app/fixtures/ExpectedDocuments/ExpectedDocuments.php

```diff
@@ -0,0 +1,566 @@
+                'active' => true,
+                'parent_id' => 'magentorootid',
+                'title' => 'VIP',
+            ],
+            [
+                'urls' => [['url' => 'women/tops-blouses.html']],
+                'path' => '/4/11',
+                'sort' => 2,
+                'active' => true,
+                'parent_id' => 4,
+                'title' => 'Tops & Blouses',
+            ],
+            [
+                'urls' => [['url' => 'men/tees-knits-and-polos.html']],
+                'path' => '/5/16',
+                'sort' => 3,
+                'active' => true,
+                'parent_id' => 5,
+                'title' => 'Tees, Knits and Polos',
+            ],
+            [
+                'urls' => [['url' => 'home-decor/bed-bath.html']],
+                'path' => '/7/23',
+                'sort' => 2,
+                'active' => true,
+                'parent_id' => 7,
+                'title' => 'Bed & Bath',
+            ],
+            [
+                'urls' => [['url' => 'sale/accessories.html']],
+                'path' => '/8/28',
+                'sort' => 3,
+                'active' => true,
+                'parent_id' => 8,
+                'title' => 'Accessories',
+            ],
+            [
+                'urls' => [['url' => 'men.html']],
+                'path' => '/5',
+                'sort' => 3,
+                'active' => true,
+                'parent_id' => 'magentorootid',
+                'title' => 'Men',
+            ],
+            [
+                'urls' => [['url' => 'women/pants-denim.html']],
+                'path' => '/4/12',
+                'sort' => 3,
+                'active' => true,
+                'parent_id' => 4,
+                'title' => 'Pants & Denim',
+            ],
+            [
+                'urls' => [['url' => 'men/pants-denim.html']],
+                'path' => '/5/17',
+                'sort' => 4,
+                'active' => true,
+                'parent_id' => 5,
+                'title' => 'Pants & Denim',
+            ],
+            [
+                'urls' => [['url' => 'home-decor/electronics.html']],
+                'path' => '/7/24',
+                'sort' => 3,
+                'active' => true,
+                'parent_id' => 7,
+                'title' => 'Electronics',
+            ],
+        ];
+    }
+
+    /**
+     * Return test Content array.
+     *
+     * @return array
+     */
+    public static function getContentDocument()
+    {
+        return [
+            [
+                'slug' => 'customer-service',
+                'title' => 'Customer Service',
+                'content' => 'Customer Service content',
+            ],
+            [
+                'heading' => 'Reward Points',
+                'slug' => 'reward-points',
+                'title' => 'Reward Points',
+                'content' => '<p>The Reward Points Program allows you to earn points for certain actions you take ' .
```

Comment:
I don't really see the point of having this wall of text in your test. I would suggest truncating this text to a minimum both here and in your database fixture.